jobs/bump-lockfile.Jenkinsfile

```diff
@@ -181,6 +188,28 @@ EOF
                 shwrap("tar -cf - tmp/kola-testiso-uefi/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-uefi.tar.xz")
                 archiveArtifacts allowEmptyArchive: true, artifacts: 'kola-testiso*.tar.xz'
             }
+        }, s390x: {
```

Comment:
we should maybe move the x86_64 section to the top or the bottom so the aarch64 and s390x sections can be next to one another. If you move it to the top maybe do it in a separate commit prep commit.

Reply:
addressed in the recent commit.